pages/index.tsx

```diff
@@ -137,6 +164,22 @@ export const HomePage: NextPage = () => {
       </Stack>
       <Stack spacing="32px" px="112px" py="64px">
         <Button onClick={onDrawerOpen}>View your list</Button>
+        <Select
+          isMulti
+          isSearchable
+          options={allNeighborhoods
+            ?.map((neighborhood) => ({
+              value: neighborhood,
+              label: neighborhood['Neighborhood Name'],
+            }))
+            .sort(function (a, b) {
+              return a.label.localeCompare(b.label)
+            })}
+          onChange={(values) =>
+            setSelectedNeighborhoods(values.map((value) => value.value))
+          }
+          placeholder="Filter by neighborhood"
+        />
```

Comment:
Not anything pressing but we have some styles for ``Select`` components throughout the codebase maybe we should just create a single styled ``Select`` wrapper so that we keep the styling the same

Reply:
Good call out - maybe we can do in a dedicated separate PR?